evaluation/evaluate.py

```diff
@@ -194,81 +196,224 @@ def run_evaluation_from_config(working_dir: Path, config: dict, num_questions: i
         logger.error("Evaluation was terminated early due to an error ⬆")
 
 
+def plot_bar_charts(
+    layout: Tuple[int, int],
+    data: List[Dict[str, any]],
+    titles: List[str],
+    y_labels: List[str],
+    y_max_lim: List[float] = None,
+    width=0.4,
+    output_path: Path = Path("evaluation_results.png"),
+):
+    """Plot bar charts for the provided data."""
+    assert layout[0] * layout[1] == len(data), "Number of data points must match the layout"
```

Comment:
We don't typically raise `AssertionErrors` outside of tests when we can have much richer errors. I'd probably switch to sth like this:
```suggestion
    if layout[0] * layout[1] != len(data):
        raise ValueError("Number of data points must match the layout")
```

Reply:
sure that makes sense, I'll change them in my next commit along with other fixes.

Reply:
Note that suggestions like that can be committed directly from the GH UI!

Reply:
bug: layout is fixed to (1, 3), if we only select 2 of the numeric da, it will raise the Error. mark here, will take a look how to fix this later